ansible/roles/providers/tasks/vmware.yml

```diff
@@ -58,6 +58,14 @@
     dest: /tmp/cloud-init-vmware.sh
     mode: 0700
 
+# pip on CentOS needs to be upgraded, but since it's still
+# Python 2.7, need < 21.0
+- name: Upgrade pip
+  pip:
+    name: pip<21.0
+    state: forcereinstall
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version == '7'
```

Comment:
This needs to handle offline case similar to https://github.com/mesosphere/konvoy-image-builder/blob/dba46aebd4abd806621f30a9c9054f7f25033cd6/ansible/roles/providers/tasks/aws.yml#L22

Reply:
I have a separate story coming up next sprint to cover all offline use cases. I still have to work getting this deployed using vsphere "air gapped" VM network. 
but i will fix this in this PR.

Reply:
>but i will fix this in this PR.
No need we will want to add tests anyways to verify it, lets tackle this in the next iteration.